includes/EDD_SL_Plugin_Updater.php

```diff
@@ -260,7 +260,7 @@ public function plugins_api_filter( $_data, $_action = '', $_args = null ) {
 
 		$cache_key = 'edd_api_request_' . md5( serialize( $this->slug . $this->api_data['license'] ) );
 
-		//Get the transient where we store the api request for this plugin for 24 hours
+		// Get the transient where we store the api request for this plugin for 24 hours
 		$edd_api_request_transient = get_site_transient( $cache_key );
```

Comment:
@pippinsplugins Shouldn't this have been replaced as well? We're not setting this anymore, so I'm guessing that'll always return nothing.